From 51685a4b267ae1775d3679ba31ee9dcb0206ca25 Mon Sep 17 00:00:00 2001
From: Alex Vandiver <alex@chmrr.net>
Date: Fri, 29 Jul 2016 17:47:28 -0700
Subject: [PATCH 5/5] Fix tests for Mojolicious 7.0

Mojolicious 7.0 "removed support for smart whitespace trimming from
all_text and text methods in Mojo::DOM."  This causes tests to fail.

Adjusts tests to work in both versions, via a regex.
---
 t/web/cf_groupings.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/web/cf_groupings.t b/t/web/cf_groupings.t
index 0a40f71..03ca60a 100644
--- a/t/web/cf_groupings.t
+++ b/t/web/cf_groupings.t
@@ -65,7 +65,7 @@ my $id = $m->get_ticket_id;
     foreach my $grouping (@groupings) {
         my $row_id = "CF-$CF{$grouping}-ShowRow";
         is $dom->find(qq{#$row_id})->size, 1, "CF on the page";
-        is $dom->at(qq{#$row_id})->all_text, "Test$grouping: Test${grouping}Value", "value is set";
+        like $dom->at(qq{#$row_id})->all_text, qr/Test$grouping:\s*Test${grouping}Value/, "value is set";
         ok $dom->at(qq{$location{$grouping} #$row_id}), "CF is in the right place";
     }
 }
-- 
2.9.3

